documentation/development.md

```diff
@@ -122,6 +122,44 @@ LOG_LEVEL=Debug
 OPERATOR_NAME=ide
 ```
 
+#### Deploy using the Helm Charts
+This section provides an instructions for running the operator Helm charts with an image that is built from the local branch.
+
+Build the image from the current branch.
+
+```
+$ export DOCKER_REPO_BASE=jsanda
```

Comment:
I propose to use `orangeopensource` here. Anyone can change it to the account they want but that's the one we use to store our images for instance